modules/mobile/app/models/mobile/v0/adapters/vaos_v2_appointments.rb

```diff
@@ -49,13 +49,27 @@ class VAOSV2Appointments
 
         # Only a subset of types of service that requires human readable conversion
         SERVICE_TYPES = {
-          outpatientMentalHealth: 'Mental Health',
-          moveProgram: 'Move Program',
-          foodAndNutrition: 'Nutrition and Food',
-          clinicalPharmacyPrimaryCare: 'Clinical Pharmacy Primary Care',
-          primaryCare: 'Primary Care',
-          homeSleepTesting: 'Home Sleep Testing',
-          socialWork: 'Social Work'
+          'amputation' => 'Amputation care',
+          'audiology' => 'Audiology and speech (including hearing aid support)',
+          'audiology-routine exam' => 'Routine hearing exam',
+          'CCAUDRTNE' => 'Routine hearing exam',
```

Comment:
I deviated from the table listed in the description based on some test data I found that was using the ccid (ALL CAPS ones) instead of the idv2 like VAOS was saying it should be. Still waiting to hear back on if this is still a current concern since the test data was back from 2019 but we may not get that sorted until after v2 release so I'm putting out the PR now but can remove the ccids later if we find they aren't necessary.